packages/circuit-ui/components/NotificationInline/NotificationInline.docs.mdx

```diff
@@ -0,0 +1,35 @@
+import { Status, Props, Story } from '../../../../.storybook/components';
+import { NotificationToast } from '@sumup/circuit-ui';
+
+# NotificationInline
+
+<Status.Stable />
+
+<Story id="notification-notificationinline--base" />
+<Props />
+
+The Notification Inline component non-disruptively provides quick and contextual inline notification and it is integrated within the content section.
+Inline notifications are frequently used to provide additional guidance to the user, like positive feedback, confirmation about an action or errors in forms, cards or dialog boxes.
+
+## Usage guidelines
```

Comment:
I think I mentioned this already, but I've been moving away from these vague "usage guidelines" in the docs I've written lately. I think the information is valuable but could be better organized, so for example:

- details about props ("provide body copy", "optional headline or action button"): these can be removed altogether, and we can just make sure to have enough stories that show the different combinations
- accessibility guidelines ("be concise, specific in copy", "place near the content"): this could go in an _Accessibility_ section with reference to WCAG SCs
- variant-specific guidelines ("give clear next steps" when requiring user action): we could add these under the variant itself

So basically, I think that everything we've been putting under "usage guidelines" could be either removed or put elsewhere, in a clearer way—right no it's not obvious to me what these are actually meant for, since they're a pretty subjective mix of guidelines

Reply:
Thanks a lot. I will adjust it and move the guidelines to the respective sections then.

Reply:
Updated the docs, and removed the parts from the Usage guidelines to different sections. Let me know if you think we should do some more changes. Thanks.